cluster-operator/src/main/java/io/strimzi/operator/cluster/model/PodDisruptionBudgetUtils.java

```diff
@@ -51,39 +51,6 @@ public static PodDisruptionBudget createPodDisruptionBudget(
                 .build();
     }
 
```

Comment:
No need for the double empty lines here which happen after you delete the method.